internal/dispatch/graph/check_test.go

```diff
@@ -216,16 +216,16 @@ func TestCheckMetadata(t *testing.T) {
 				ONR("user", "product_manager", graph.Ellipsis),
 				[]expected{
 					{"owner", true, 1, 1},
-					{"edit", true, 2, 2},
-					{"view", true, 3, 3},
+					{"edit", true, 3, 2},
```

Comment:
Why did the dispatch count increase for requests that didn't have debug data?

Reply:
I fixed the use of the response metadata in the `union` call in Check: it now uses the combined metadata, rather than just the subrequest one